src/canvas/Canvas.ts

```diff
@@ -8,6 +8,7 @@ import type {
   Transform,
 } from '../EventTypeDefs';
 import { Point } from '../Point';
+import { ActiveSelection } from '../shapes/ActiveSelection';
```

Comment:
i want to avoid the library to be a monoblock unsplittable, so please we need to do a function that identify this being an active selection without importing it.